app/views/govuk_publishing_components/components/_tabs.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  tabs ||= []
+%>
+<% if tabs.count > 1 %>
+  <div class="govuk-tabs gem-c-tabs" data-module="tabs">
+    <h2 class="govuk-tabs__title">
+      Contents
+    </h2>
+    <ul class="govuk-tabs__list">
+      <% tabs.each do |tab| %>
+      <li class="govuk-tabs__list-item">
+        <a class="govuk-tabs__tab" href="#<%= tab[:id] %>">
+            <%= tab[:title] %>
+          </a>
+      </li>
+      <% end %>
+    </ul>
+    <% tabs.each do |tab| %>
+      <section class="govuk-tabs__panel" id="<%= tab[:id] %>">
+        <h2 class="govuk-heading-l"><%= tab[:title] %></h2>
+        <%= raw tab[:content] %>
```

Comment:
Is there a way to avoid using 'raw', which could introduce XSS risk?

Reply:
Do you have any suggestions? I'm expecting to be passing in tables, divs etc so I need to be able to render html content, and I'm not sure how I'd pass more than one block to a single component - if you have examples I'm all ears.

Reply:
I think @tijmenb removed the other instances of raw, forcing the user of the component to use a 'capture' to pass into the tab

If there's a single place for nested children - use 'yield' see https://github.com/alphagov/govuk_publishing_components/pull/356 for an example

If there's multiple slots - use

```
<% tabOneHtml = capture do %>
  <h1>Content for tab 1</h1>
<% end %>
<% tabTwoHtml = capture do %>
  <h1>Content for tab 1</h1>
<% end %>

<%= render "govuk_publishing_components/components/tabs", {
  tabs: [
    {
      id: "tab1",
      title: "First section",
      content: tabOneHtml
    },
    {
      id: "tab2",
      title: "Second section",
      content: tabTwoHtml
    }
  ]
} %>

```

Or (if the data is passed and sanitised from a presenter/controller

```

<%= render "govuk_publishing_components/components/tabs", {
  tabs: [
    {
      id: "tab1",
      title: "First section",
      content: data.tabOneHtml.html_safe
    },
    {
      id: "tab2",
      title: "Second section",
      content: data.tabTwoHtml.html_safe
    }
  ]
} %>
```

Reply:
Yep! That looks good.

Reply:
Thanks.